src/query/expression/src/evaluator.rs

```diff
@@ -249,6 +266,12 @@ impl<'a> Evaluator<'a> {
 
                 self.run_lambda(name, args, lambda_expr, return_type)
             }
+            // async function call values has been generated by transformer
+            Expr::AsyncFunctionCall { .. } => {
```

Comment:
The  `Expr::AsyncFunctionCall`  will be replaced by `Expr::ColumnRef`. There is no need to introduce `Expr::AsyncFunctionCall` in evaluator.